backend/src/controllers/mongodb/transactions.ts

```diff
@@ -0,0 +1,103 @@
+import {Request, Response} from 'express';
+import { TransactionModel } from '../../models/transaction.js';
+import { TransactionDAO, TransactionStatus } from '../../entities/transaction.js';
+import { ProductModel } from '../../models/product.js';
+import { ProductDAO } from '../../entities/product.js';
+
+export async function getTransactions(req: Request, res: Response) {
+    try{
+        const orders = await TransactionModel.find();
+        res.status(201).send(orders);
+    } catch (error) {
+        res.status(500).send({
+            type: "urn:100-project:error:unable_to_get_transactions",
+            title: "Unable to get transactions",
+            status: 500,
+            detail: "An error occured while fetching transactions"
+        })
```

Comment:
Wag sobrang specific nung type and title. Pwedeng `urn:100-project:error:internal_server_error` and `"Internal server error"` nalang yung type and title.